readme.md

```diff
@@ -41,6 +41,7 @@ Our hope is that GitHub will notice and implement some of these much needed impr
 - Removes tooltips
 - Copy canonical link to file when [the `y` hotkey](https://help.github.com/articles/getting-permanent-links-to-files/) is used
 - ~~[Adds blame links for parent commits in blame view](https://github.com/sindresorhus/refined-github/issues/2#issuecomment-189141373)~~ [Implemented by GitHub](https://github.com/blog/2304-navigate-file-history-faster-with-improved-blame-view)
+- [Linkifies URLs and issue references in code](https://cloud.githubusercontent.com/assets/737065/25365902/f407f964-2939-11e7-9c66-7d48f31c5576.png)
```

Comment:
Add it much closer to the top. Around line 20. This feature is awesome.

Maybe we should split it into two feature highlights?

1. Linkifies issue references in code comments
2. Linkifies URLs in code and comments

^ Or something better.